examples/sampling/pyg/node_classification.py

```diff
@@ -120,7 +145,7 @@ def create_dataloader(dataset_set, graph, feature, device, is_train):
     return dataloader
 
 
-def train(model, dataloader, optimizer, criterion, device, num_classes):
+def train(model, dataloader, optimizer, num_classes):
     #####################################################################
```

Comment:
Too many unnecessary HIGHLIGHT, the only highlight in this example is to "highlight how to adopt GB with PYG"